thewalrus/_hermite_multidimensional.py

```diff
@@ -203,10 +203,10 @@ def hermite_multidimensional_numba(R, cutoff, y, C=1, dtype=None):
     if y.shape[0] != n:
         raise ValueError(f"The matrix R and vector y have incompatible dimensions ({R.shape} vs {y.shape})")
     num_indices = len(y)
-    if isinstance(cutoff, int):
-        cutoffs = tuple([cutoff] * num_indices)
-    else:
+    if isinstance(cutoff, (list, tuple)):
```

Comment:
This could be made slightly safer by checking if `cutoff` is an iterable instead of just a list or a tuple (might need to import `Iterable` from `collections.abc` as well).
```suggestion
    if isinstance(cutoff, Iterable):
```